amqtt/plugins/topic_checking.py

```diff
@@ -66,11 +67,24 @@ async def topic_filtering(self, *args, **kwargs):
         if filter_result:
             session = kwargs.get("session", None)
             req_topic = kwargs.get("topic", None)
+            action = kwargs.get("action", None)
+
+            # hbmqtt and older amqtt do not support publish filtering
+            if (action == "publish") and ("publish-acl" not in self.topic_config):
```

Comment:
```suggestion
            if action == "publish" and "publish-acl" not in self.topic_config:
```

the parentheses are not needed

Reply:
Heh, blame me bouncing between C, C++, JavaScript, Java and Python thus not always remembering operator precedence and basically brute-forcing my intentions… I'll clean that up. :-)